File: FloridaShiduch/Scripts/src/models/bindings/personal.js
define(['bindings', 'behaviors/personal'], function (Bindings, PersonalBehavior) {
    'use strict';
    // Singleton
    var bindings = new Bindings({
        '@ui.birthday': {
            observe: 'birthday',
            setOps: { validate: true }
        },
        '@ui.feet': {
            observe: 'feet',
            setOps: { validate: true }
        },
        '@ui.inches': {
            observe: 'inches',
            setOps: { validate: true }
        },
        '@ui.build': {
            observe: 'build',
            setOps: { validate: true }
        },
        '@ui.maritalStatus': {
            observe: 'maritalStatus',
            setOps: { validate: true }
        },
        '@ui.children': {
            observe: 'children',
            setOps: { validate: true }
        },
        '@ui.childrenNumber': {
            observe: 'childrenNumber',
            setOps: { validate: true }
        },
        '@ui.pets': {
            observe: 'pets',
            setOps: { validate: true }
        },
        '@ui.smoke': {
            observe: 'smoke',
            setOps: { validate: true }
        }
    }, PersonalBehavior.prototype.ui);


    return bindings;
});

File: FloridaShiduch/Models/Profile/Personal.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Web;

namespace FloridaShiduch.Models.Profile
{
    public class Personal
    {
        [Key, ForeignKey("ApplicationUser")]
        [JsonProperty("userId")]
        public string UserId { get; set; }

        [JsonProperty("birthday")]
        public DateTime Birthday { get; set; }

        [JsonProperty("feet")]
        public int Feet { get; set; }

        [JsonProperty("inches")]
        public int Inches { get; set; }

        [JsonProperty("build")]
        public string Build { get; set; }

        [JsonProperty("maritalStatus")]
        public string MaritalStatus { get; set; }

        [JsonProperty("children")]
        public bool Children { get; set; }

        [JsonProperty("childrenNumber")]
        public int ChildrenNumber { get; set; }

        [JsonProperty("pets")]
        public bool Pets { get; set; }

        [JsonProperty("smoke")]
        public bool Smoke { get; set; }

        public virtual ApplicationUser ApplicationUser { get; set; }
    }
}